codegen/src/test/kotlin/tools/samt/codegen/http/HttpTransportTest.kt

```diff
@@ -267,6 +267,40 @@ class HttpTransportTest {
         parseAndCheck(source to listOf("Error: No operation with name 'bar' found in service 'Greeter' of provider 'GreeterEndpoint'"))
     }
 
+    @Test
+    fun `cannot map two operations to the same method path combination`() {
+        val source = """
+            package tools.samt.greeter
+
+            service Greeter {
+                greet(name: String): String
+                greetTwo(name1: String, name2: String): String
+                greetThree(name: String): String
+                greetFour(name: String): String
+            }
+
+            provide GreeterEndpoint {
+                implements Greeter { greet, greetTwo, greetThree, greetFour }
+
+                transport http {
+                    operations: {
+                        Greeter: {
+                            greet: "GET /greet",
```

Comment:
To be ultra-correct, I would also add a test where you do not specify an operation configuration explicitly

Reply:
By default this would set the path to `/$operationName`, which would be unique within services. However if multiple services have the same operationName this would cause an error, as you mentioned in your other comment. I will investigate